frontend/src/features/admin/admin-areas/interfaces.ts

```diff
@@ -0,0 +1,39 @@
+/** interface containing the structure of data found in administrative areas */
+export interface IAdministrativeArea {
```

Comment:
Make a separate file for each interface so that it's easy to find them.

Reply:
If you want to keep everything clean, throw them in a folder called `interfaces` and add an `index.ts` file that points to each one.  A little extra boilerplate, but it makes it nice and clean.